archlinux/content.md

```diff
@@ -0,0 +1,32 @@
+# What is Arch Linux?
+
+Arch Linux, is a lightweight and flexible Linux® distribution that tries to Keep It Simple.
+
+Currently we have official packages optimized for the x86-64 architecture. We complement our official package sets with a community-operated package repository that grows in size and quality each and every day.
+
+Our strong community is diverse and helpful, and we pride ourselves on the range of skillsets and uses for Arch that stem from it. Please check out our forums and mailing lists to get your feet wet. Also glance through our wiki if you want to learn more about Arch.
```

Comment:
It might be a good idea to turn "our wiki" here into a link to the (unparalleled) Arch wiki so the uninitiated can find it easier. :smile:

Reply:
Good point! I'll add the link.